src/Slider/index.jsx

```diff
@@ -207,6 +212,67 @@ export default class Slider extends Component
         this.handleMouseUp   = this.handleMouseUp.bind( this );
     }
 
+    componentDidMount()
+    {
+        this.attachInputRefs();
+    }
+
+    componentWillUpdate( nextProps )
+    {
+        if ( nextProps.inputRef !== this.props.inputRef )
+        {
+            this.detachInputRefs();
+        }
+    }
+
+    componentDidUpdate( prevProps )
+    {
+        const { props } = this;
+
+        if ( prevProps.inputRef !== props.inputRef ||
+            prevProps.value.length !== props.value.length )
+        {
+            this.attachInputRefs();
+        }
+    }
+
+    componentWillUnmount()
+    {
+        this.detachInputRefs();
+    }
+
```

Comment:
Looking great! Just 2 minor things: it would be good to put here `/* eslint-disable react/sort-comp */` so that linter stops complaining 😄 ; and on line 258 there's semicolon missing.